README.md

```diff
@@ -109,6 +110,17 @@ displayed.
 
 NOTE: PWD hooks are currently not supported for POSIX shells.
 
+#### PowerShell
+
+Add the following line to your profile:
+
+```powershell
+Invoke-Expression (& {
+    $hook = if ($PSVersionTable.PSVersion.Major -lt 6) { 'prompt' } else { 'pwd' }
```

Comment:
Why not always default to `prompt`? This will ensure that the PowerShell implementation stays consistent with the other shells.

Reply:
I don't think consistency should matter if there is a better way, and running `zoxide` less frequently seems better to me. I also think few know about the `pwd` hook and it would go unused otherwise.

Also - placement of this snippet matters more for the `prompt` hook as users typically define their own prompt but do not use the `pwd` hook. This snippet relies on being placed after a previously defined `prompt` function.

Reply:
We document the hooks we support [in the README](https://github.com/ajeetdsouza/zoxide#init-flags). Considering the section documenting this is right below your new PowerShell documentation, I don't think this is a good justification.

Why does "running `zoxide` less frequently" sound better to you? Although I was the original proponent for a PWD hook, I haven't actually used it since it was implemented.

Placement of the snippet is not something we can control. That's why you see the `posix` hook adding to the user's `PS1`. We can't prevent the user from overwriting the hook.

If a user opts to use the `prompt` hook, then we still have the same problem where it must be placed after a previously defined `prompt` function.

Reply:
I wasn't aware of the `pwd` hook in any shell until I had started implementing this PR. I didn't even read all of the README, so I expect plenty of people to miss that.

I also didn't realize PowerShell had this hook since it is new, so I think it will be easy to miss and easy for people to assume you must modify your prompt, which in turn will turn some people away.

As for running less frequently - I know it's fast, but on Windows, especially in corporate domains - sometimes even fast things can be slow. It might be an IT department logging that you can't disable, or anti-virus or whatever, so running less often is just wiser IMO.

Reply:
I do see what you mean, and I myself have been using the pwd hook on my system ever since it was implemented. However, I'd still like to keep the behaviour uniform - simply because it is less surprising to the user.

Assuming we default to a pwd hook unless it is unavailable, I can think of an example where this might be a problem - say, a user uses both bash and a POSIX shell on his system and initializes zoxide on them. Since the POSIX shell does not have a pwd hook, we default to the prompt hook. In this case, entries created by the POSIX shell will completely overshadow the entries created by bash, due to the frequent updates. Another (minor) issue I can think of would be that interactive queries would show much greater frecencies on some systems, without much of an explanation.

Admittedly, these are contrived examples, but I hope you see my point. I think requiring the user to explicitly enable pwd hooks ensures that they know what they're doing, and prevents confusion.

I can think of two solutions to this:
1. We could add a line to the README stating that users who need performance should go for pwd hooks.
2. We could default to pwd hooks (breaking change), and fail if they are unavailable with a help message:

```
PWD hooks are currently not available for this shell. Consider adding `--hook prompt` to your `init` command to fix this.
```

Reply:
@lzybkr I've changed the default hook to PWD in https://github.com/ajeetdsouza/zoxide/commit/7eed1ed9c3ac3eec27896329a05d98ee3ee9d012. Could you change this line to use the PWD hook by default, but explicitly require `--hook prompt` if it is unavailable?

Reply:
@ajeetdsouza - I believe this implements exactly what you're asking for. The `pwd` hook is only available in PowerShell versions 6 or greater.